### ISSUE TITLE
caddy plugin

### ISSUE THREAD

**Issue by Ogy** [Snipa22/nodejs-pool#236]
Hi @Snipa22 can you please add at least http.ipfilter to your caddy ?
Thanks for your work and time

**Comment by camthegeek** [comment]
You can change your caddyfile to add whatever you like. It would be pointless to add a filter into the default caddyfile as each server and pool operator comes from a different IP.

**Comment by Ogy** [comment]
yeah i can change caddyfile, but there are plugins for caddy and snipas hosted caddy.tar.gz doesnt have http.ipfilter included
i'd like to filter /admin
